package exceptiontopic;

public class TestApp1 
{
	public static void main(String[] args) 
	{
//		int a[] = new int[5];//a[0],a[1],a[2],a[3],a[4]
//		a[5] = 100;// raise Exception----"ArrayIndexOutofBoundException"
//		System.out.println(a[5]);
//     -----------------------------------------------------------------------------
//		String value = "10A43";	
//		int no = Integer.parseInt(value);// "NumberFormateException"
//		System.out.println(no);// "10 integer values"
//	     -----------------------------------------------------------------------------
//		String name = "royaltechnosoft";
//		//     0123456789012345
//		System.out.println(name.length());//15
//		System.out.println(name.charAt(name.length()));//"StringIndexoutofBoundException"
//		
//		System.out.println("Statement---6 ");
//		System.out.println("Statement---7 ");
//		System.out.println("Statement---8 ");
//		System.out.println("Statement---9 ");
//		System.out.println("Statement---10");
//	     -----------------------------------------------------------------------------
//		int no1,no2,ans;
//		no1 = 10;
//		no2 = 0;
//		ans = no1/no2;// "ArithmeticException"
//		System.out.println("Addition : " + ans);
//	     -----------------------------------------------------------------------------
		String name = null;
		System.out.println(name.length());
		
	}
}
